src/server/sproutVideo/video.js

```diff
@@ -8,7 +11,31 @@ class Video {
   }
 
   get(id) {
-    return this._request.get(`videos/${id}`);
+    return this._request.get(`videos/${id}`)
+      .then(res => camelCaseKeys(res, { deep: true }))
+      .then(({ selectedPosterFrameNumber, ...rest }) => ({
+        ...rest,
+        selectedPosterFrameIndex: selectedPosterFrameNumber
+      }));
+  }
+
+  async updatePosterFrame(id, { customPosterFrame: content, posterFrameIndex }) {
```

Comment:
Why do we need to rename `customPosterFrame`?

Reply:
We don't 😄 It was just more clear to me. Should I keep `customPosterFrame` instead?